action.yml

```diff
@@ -26,6 +26,9 @@ inputs:
     description: 'The root folder for the theme files that will be uploaded (default: ".")'
     required: false
     default: '.'
+  pull_theme:
+    description: 'If provided, the settings and json templates will be pulled into the test theme.'
```

Comment:
The description made me feel like this was a boolean, can we be more explicit about the type of the expected value?

Something like:

```
'The ID of a theme from which the settings and JSON templates should be pulled. If provided, those settings will be pulled into the development theme.'
```

Also, can this be added to the README as well?